build/tools/templates/Archive/Archive.csproj

```diff
@@ -5,6 +5,8 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <EnableApiCheck>false</EnableApiCheck>
+    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
```

Comment:
+💯. Thanks for adding this

Reply:
I think we should investigate whether we want to have this in other templates as well.